### DISCUSSION TITLE
Styled-Components transient props

### DISCUSSION THREAD

**Discussion by alex84G** [react-dropzone/react-dropzone#1343] [Q&A]
Hi,
I use the Styled-Components example to style the dropzone: https://react-dropzone.js.org/#using-styled-components

Now, I'am getting following errors:

![grafik](https://github.com/react-dropzone/react-dropzone/assets/2656031/82d7fbb9-5350-4979-bfec-75d82f168c1b)

I know, we can use the Transient props of Syteld-Components to fix this error: 
https://styled-components.com/docs/api#transient-props

But I don't know how to use the transient props in the getRootProps() method of dropzone:

<Dropzone {...getRootProps({ isFocused, isDragAccept, isDragReject })}>

Can someone help me?

**Comment by rolandjitsu** [comment]
@alex84G does https://react-dropzone.js.org/#using-styled-components help?
